tools/cli/pkg/command/runtime.go

```diff
@@ -177,8 +177,8 @@ func (cmd *RuntimeCommand) Validate() error {
 	if cmd.opDetail {
 		cmd.params.OperationDetail = runtime.AllOperation
 	}
-	if cmd.params.OnlyDeleted == true && len(cmd.params.InstanceIDs) == 0 {
-		return fmt.Errorf("need to provide some Instance IDs when using --only-deleted")
+	if slices.Contains(cmd.params.States, runtime.StateDeprovisioned) && len(cmd.params.InstanceIDs) == 0 {
+		return fmt.Errorf("must provide some Instance IDs when looking up deprovisioned runtimes")
```

Comment:
this limitation remains because deprovisioned runtimes are reconstructed best effort from the `operations` table and which can contain a lot of historical data.